Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx

```diff
@@ -16,6 +16,7 @@
  *
  *=========================================================================*/
 
+#include <cstdint>
```

Comment:
Maybe `Modules/Core/Common/include/itkIntTypes.h` could be used here?

Reply:
Also, `uint8_t` is not used in this file, so this include should probably go into the header where the use occurs.

Reply:
Hello, I made the minimum change necessary to build v5.3.0 on Arch Linux, I'll check adding itkIntTypes.h


```
[ 90%] Built target ITKIOBMP-all
In file included from /home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:19:
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:41:8: warning: elaborated-type-specifier for a scoped enum must not use the 'class' keyword
   41 |   enum class Algorithm : uint8_t
      |   ~~~~ ^~~~~
      |        -----
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:41:14: error: use of enum 'Algorithm' without previous declaration
   41 |   enum class Algorithm : uint8_t
      |              ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:41:26: error: 'uint8_t' was not declared in this scope
   41 |   enum class Algorithm : uint8_t
      |                          ^~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:23:1: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   22 | #include "ITKMathematicalMorphologyExport.h"
  +++ |+#include <cstdint>
   23 | 
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:42:3: error: default member initializer for unnamed bit-field
   42 |   {
      |   ^
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h:52:107: error: 'Algorithm' in 'class itk::MathematicalMorphologyEnums' does not name a type
   52 |                                         operator<<(std::ostream & out, const MathematicalMorphologyEnums::Algorithm value);
      |                                                                                                           ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:25:67: error: 'Algorithm' in 'class itk::MathematicalMorphologyEnums' does not name a type
   25 | operator<<(std::ostream & out, const MathematicalMorphologyEnums::Algorithm value)
      |                                                                   ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx: In lambda function:
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:30:41: error: 'itk::MathematicalMorphologyEnums::Algorithm' has not been declared
   30 |       case MathematicalMorphologyEnums::Algorithm::BASIC:
      |                                         ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:32:41: error: 'itk::MathematicalMorphologyEnums::Algorithm' has not been declared
   32 |       case MathematicalMorphologyEnums::Algorithm::HISTO:
      |                                         ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:34:41: error: 'itk::MathematicalMorphologyEnums::Algorithm' has not been declared
   34 |       case MathematicalMorphologyEnums::Algorithm::ANCHOR:
      |                                         ^~~~~~~~~
/home/fabio/Dev/AURNEW/insight-toolkit/src/InsightToolkit-5.3.0/Modules/Filtering/MathematicalMorphology/src/itkMathematicalMorphologyEnums.cxx:36:41: error: 'itk::MathematicalMorphologyEnums::Algorithm' has not been declared
   36 |       case MathematicalMorphologyEnums::Algorithm::VHGW:
      |                                         ^~~~~~~~~
[ 90%] Built target ITKIOBioRad-all
[ 90%] Built target ITKIOBruker-all
make[2]: *** [Modules/Filtering/MathematicalMorphology/src/CMakeFiles/ITKMathematicalMorphology.dir/build.make:76: Modules/Filtering/MathematicalMorphology/src/CMakeFiles/ITKMathematicalMorphology.dir/itkMathematicalMorphologyEnums.cxx.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:6002: Modules/Filtering/MathematicalMorphology/src/CMakeFiles/ITKMathematicalMorphology.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 90%] Built target ITKIOCSV-all
```

Reply:
Yes, `Modules/Core/Common/include/itkIntTypes.h` can be used, force pushed to use that